src/RewardsStreamerMP.sol

```diff
@@ -54,6 +56,7 @@ contract RewardsStreamerMP is
 
     struct VaultData {
         uint256 stakedBalance;
+        uint256 mpStaked;
```

Comment:
I think this is problematic and will likely corrupt storage of the already existing stake manager.
This likely needs to be moved to the end of the struct

Reply:
Also, if I'm not mistaken `mpStaked` is what `mpAccrued` used to be no?

`mpAccrued` where the "settled" MPs. The same ones that users will settle when they claim/stake their live MPs with the changes to come.

Why aren't we just renaming `mpAccrued` then (or stick with `mpAccrued`)?

Reply:
> Also, if I'm not mistaken mpStaked is what mpAccrued used to be no?

mpAccrued are the MPs accrued over time and snapshotted every time the shares change. 

mpStaked are the MP that become part of the shares/weights. initialMP + bonus are staked automatically. other MPs will be staked mabnually with the boost/compound function